lib/src/config/constants.dart

```diff
@@ -0,0 +1,7 @@
+class AppConstants {
+  static const String ENV = 'ENV';
+  static const String HOST_NAME = 'HOST_NAME';
+  static const String DEFAULT_ENV = 'coin';
```

Comment:
I think ENV should be Prod/Dev/Uat

Reply:
You can explain ENV for the first or last or both. I will change it.

Reply:
Can I change to become this code?

Reply:
static const String ENV = 'PROD';